Makefile

```diff
@@ -32,6 +32,50 @@ else ifeq ($(OS_GENERAL),Windows)
 	OS_PACKAGE_MANAGER := choco
 endif
 
+# Detect the operating system
+UNAME_S := $(shell uname -s)
```

Comment:
todo: We already have this I believe in `OS_GENERAL`

Reply:
changed